src/tests/unit/gna/gna_export_import_test.cpp

```diff
@@ -162,11 +172,29 @@ TEST_F(GNAExportImportTest, ExportImportI16) {
 }
 
 TEST_F(GNAExportImportTest, ExportImportI8) {
-    ov::AnyMap gna_config = {
+    gna_config = {
         ov::intel_gna::execution_mode(ov::intel_gna::ExecutionMode::SW_EXACT),
         ov::hint::inference_precision(ngraph::element::i8)
     };
     exported_file_name = "export_test.bin";
     ExportModel(exported_file_name);
     ImportModel(exported_file_name);
 }
+
+TEST_F(GNAExportImportTest, HideLibVersionFromModelInLogNoMode) {
+    gna_config = {ov::log::level(ov::log::Level::NO)};
+    EXPECT_THAT(ExportImportModelWithLogLevel(),
+        Not(HasSubstr(ov::intel_gna::common::get_openvino_version_string())));
+}
+
+TEST_F(GNAExportImportTest, HideLibVersionFromModelInLogWarnMode) {
+    gna_config = {ov::log::level(ov::log::Level::WARNING)};
+    EXPECT_THAT(ExportImportModelWithLogLevel(),
+        Not(HasSubstr(ov::intel_gna::common::get_openvino_version_string())));
+}
+
+TEST_F(GNAExportImportTest, ShowLibVersionFromModelInLogDebugMode) {
+    gna_config = {ov::log::level(ov::log::Level::DEBUG)};
```

Comment:
Parametrized test may help to test all the log levels. Why do you ignore trace level?

Reply:
Only log levels: NO, WARING, DEBUG are supported by GNA plugin, for rest of log configurations GNAPlugin throws exception at the moment of object configuration (UpdateFieldsFromConfig()). The intent of these tests is not to verify supported log levels but only check that version string is printed for supported log levels.